### ISSUE TITLE
Tabs JavaScript not removing 'active' class from nav-pills nav-stacked

### ISSUE THREAD

**Issue by p-h-i-l** [twbs/bootstrap#20797]
Expected Behaviour: 
When activating a `nav-link`, the `active` state gets removed and added to the now active `nav-link`. Therefore, the active state gets removed and added to the `active` item. Additionally the content transitions between the current and now active. 

Problem:
The active class is not being removed from `nav-lin`k. Thus each item can only be navigated to once. 

Codepen ( **UPDATED** ) :
http://codepen.io/anon/pen/ALRGqj?js-preprocessor=none

I hope the syntax is correct since there is no example in the docs:

``` html
<!-- Navigation -->
<nav class="nav nav-pills nav-stacked" role="tablist">
  <a class="nav-link active" data-toggle="pill" role="tab"  href="#profile">Active</a>
  <a class="nav-link" data-toggle="pill" role="tab" href="#buzz">Link</a>
  <a class="nav-link" data-toggle="pill" role="tab" href="#references">Link</a>
  <a class="nav-link disabled" href="#">Disabled</a>
</nav>

<!-- Tab panes -->
<div class="tab-content" role="tabpanel">
  <div role="tabpanel" class="tab-pane fade in active" id="profile">...</div>
  <div role="tabpanel" class="tab-pane fade" id="buzz">bbb</div>
  <div role="tabpanel" class="tab-pane fade" id="references">ccc</div>
</div>
```

edit: **May be related to** #20039 , but tabs work, pills (stacked) dont

**Comment by twbs-lmvtfy** [comment]
Hi @p-h-i-l!

You appear to have posted a live example (http://codepen.io/anon/pen/ALRGqj.html), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 13, column 3 thru column 75: An element with ```` role=tab ```` must be contained in, or owned by, an element with ```` role=tablist ````.
* line 14, column 3 thru column 65: An element with ```` role=tab ```` must be contained in, or owned by, an element with ```` role=tablist ````.
* line 15, column 3 thru column 71: An element with ```` role=tab ```` must be contained in, or owned by, an element with ```` role=tablist ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Possible duplicate of #19374.

**Comment by mdo** [comment]
Dupe of #18566 and #19374.
